### ISSUE TITLE
setup.py try to install 'logging'

### ISSUE THREAD

**Issue by buhtz** [jessejensen/ebscopy#10]
I used the GitHub repository and tried to install ebscopy.
`sudo python3 ./setup.py install'

While this process the setup.py tried to install the `logging` package from PyPi.

  #. There is no need to do that because it commes with Python itself. It is still there.

  #. This extra-logging package crashes my complete Python system because the packge is buggy. e.g. Each call on `pip` implicite a `import logging` which fails, etc.

So please figure out why your setup.py try to install the logging package.

**Comment by nick-nz** [comment]
Yup, I can't even install this package. Have tried with Python 3.5 and 3.6 on Windows and Ubuntu systems

**Comment by bganglia** [comment]
@jessejensen I can work on this

**Comment by yuvalkry** [comment]
It is still not possible to install ebscopy in an environment with python 3 using pip.
How to bypass this?

**Comment by buhtz** [comment]
Wow, this is old.
Closing as outdated and without response from maintainer.
Please open a new issue if this is still relevant for you.
